plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -236,6 +279,47 @@ write_content_node(int64_t num_body_bytes)
   return std::string(R"(,"content":{"encoding":"plain","size":)" + std::to_string(num_body_bytes) + '}');
 }
 
+/** Initialize the generic sensitive field to be dumped. This is used instead
+ * of the sensitive field values seen on the wire.
+ */
+void
+initialize_default_sensitive_field()
+{
+  constexpr size_t default_field_size = 2048;
+  defaut_sensitive_field_value.reserve(default_field_size);
```

Comment:
Why not `resize` and then work directly on the buffer from `data()`? No need to mess about with null terminating then.

Reply:
Ah, I see - it's variable sized.

Reply:
Good idea.